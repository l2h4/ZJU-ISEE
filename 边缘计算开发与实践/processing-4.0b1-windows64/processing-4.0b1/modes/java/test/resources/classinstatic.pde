class Test {

  int getInt() { return 5; }
  String toString() {return "test";}

}


Test t = new Test();
println(t.getInt());
println(t.toString());