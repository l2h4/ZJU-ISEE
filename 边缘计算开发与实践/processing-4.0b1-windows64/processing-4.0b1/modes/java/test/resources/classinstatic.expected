import processing.core.*;
import processing.data.*;
import processing.event.*;
import processing.opengl.*;

import java.util.HashMap;
import java.util.ArrayList;
import java.io.File;
import java.io.BufferedReader;
import java.io.PrintWriter;
import java.io.InputStream;
import java.io.OutputStream;
import java.io.IOException;

public class classinstatic extends PApplet {

    public void setup() {
class Test {

  public int getInt() { return 5; }
  public String toString() {return "test";}

}


Test t = new Test();
println(t.getInt());
println(t.toString());

        noLoop();
    }

    static public void main(String[] passedArgs) {
        String[] appletArgs = new String[] { "classinstatic" };
        if (passedArgs != null) {
            PApplet.main(concat(appletArgs, passedArgs));
        } else {
            PApplet.main(appletArgs);
        }
    }
}